infrastructure/balsa_queue/bq_main_macro.hh

```diff
@@ -57,7 +58,9 @@ void run_bq(const std::string& bq_typename, const std::optional<std::string>& co
   balsa_queue.init(bq_config);
   while (!shutdown) {
     balsa_queue.base_loop();
+    jet::Timestamp loop_start_time = jet::time::get_current_time();
```

Comment:
Do we want to include `base_loop()` in the duration calculation?

Reply:
Good call. Updated.